makes/binary.mk

```diff
@@ -10,6 +10,7 @@
 # - DEPS - list of components from current repo to be completed before starting this one
 # - LIBS - names of the static libs to link the binary against (without .a suffix)
 # - LOCAL_CFLAGS  - additional CFLAGS for current component compilation
+# - LOCAL_CXXFLAGS - additional LOCAL_CXXFLAGS for current component compilation
```

Comment:
Change to:
# - LOCAL_CXXFLAGS - additional CXXFLAGS for current component compilation